CHANGES.rst

```diff
@@ -18,6 +18,17 @@ Backward incompatible changes
 - If you have made changes to the default *Monitoring Template*, then
   create a backup of your template before running migrations. Running
   migrations will make changes to the default *Monitoring Template*.
+- The time-series database schema for storing
+  `interface traffic <https://github.com/openwisp/openwisp-monitoring#traffic>`_
+  and `associated WiFi clients <https://github.com/openwisp/openwisp-monitoring#wifi-clients>`_
+  has been updated. The data for *interface traffic* and *associated WiFi clients*
+  is stored in ``traffic`` and ``wifi_clients`` measurements respectively.
+  The Django migrations will perform the necessary operations in the time-series
+  database aysnchronously. It is recommended that you backup the time-series
+  database before running the migrations.
```

Comment:
What about the fact that when the migrations are run via ansible-openwisp2, most likely the migration will be executed before supervisor is reloaded, therefore the celery task will not be available yet? What can we do about this?
Can you add a way to call the influxdb migration idempotently so we can add an explicit task in ansible-openwisp2 for this?